contributors/guide/github-workflow.md

```diff
@@ -87,9 +92,12 @@ This workflow is process-specific; for quick start build instructions for [kuber
 # While on your myfeature branch
 git fetch upstream
 git rebase upstream/master
+
+# Or
+git pull --rebase upstream master
 ```
 
-Please don't use `git pull` instead of the above `fetch` / `rebase`. `git pull`
+Please don't use `git pull` instead of the above `fetch` / `rebase`. `git pull` without any options (like `--rebase`)
```

Comment:
I think if we do want to mention `git pull --rebase`, here should be the only place, i.e. "or another non-merge options such as `git pull --rebase`".